### PR TITLE
updated alert component and badge component

### PR DESCRIPTION
**Pull request by Nagashree01**
- Added alert component: primary alert , information-alert, success-alert, error-alert,warning-alert
alert-comp

- added badge component:badge on icon and avata
badge-comp
r

### CODE REVIEW

File: doc.css [hunk 1]
```diff
@@ -47,23 +49,39 @@
...
+}
+.sub-heading{
+    font-size:var(--text-m);
+    padding: 0.5rem;
+}
+
+/* .component-view{
+    width
+} */
+
```
Comment (mariyasada) [inline doc.css:80]: remove unnecessary comment

**Review by mariyasada — APPROVED** [review]
remove unnecessary space

File: doc.css (hunk 1 above)
Comment (Nagashree01) [reply doc.css:80]: yes I'll do this

File: components/badge.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+.badge{
+    position: relative;
+    font-size: 3rem;
+
+}
+
+.badge-number{
+    position: absolute;
+    border-radius: 50%;
```
Comment (dev-enforced) [inline components/badge.css:12]: You can rename this as a variable in your base.css file as border-rd-circle if you are using this value frequently.

File: stylesheets/navbar.css [hunk 3]
```diff
@@ -8,6 +8,7 @@
     width:100%;
 }
 
+
 .navigation ul {
```
Comment (dev-enforced) [inline stylesheets/navbar.css:12]: You can use this as a utility class in your reset file mentioned as ul{list-style:none} if you wish to apply this to all the list elements which you use frequently.

**Review by dev-enforced — APPROVED** [review]
The documentation of the above components looks good.Great work @Nagashree01 . You can merge it. Suggested some minor changes which you can look on later.

File: components/badge.css (hunk 2 above)
Comment (Nagashree01) [reply components/badge.css:12]: yes, It will be very helpful as I will be using this one more frequently. Thank you.

**Pull request merged**